omnigibson/objects/dataset_object.py

```diff
@@ -323,7 +308,9 @@ def _update_texture_change(self, object_state):
             # else:
             #     print(f"Warning: DatasetObject [{self.prim_path}] does not have texture map: "
             #           f"[{target_texture_path}]. Falling back to directly updating albedo value.")
-            self._update_albedo_value(object_state, material)
+
+            if not material.is_glass:
+                self._update_albedo_value(object_state, material)
```

Comment:
So glass objects won't be able to support object-state-based texture changes? Maybe add a comment here?

Reply:
maybe update glass color attr instead

Reply:
done